scaffolds/cpp/Makefile

```diff
@@ -0,0 +1,32 @@
+cxxtestgen := cxxtestgen
+
+src = src
+src-files := \
+  $(src)/module.cpp
+
+header-files := \
+  $(src)/module.h
+
+test := test
+test-file := $(test)/tests.h
+
+build = build
+test-runner-src := $(build)/test-runner.cpp
+test-runner-exe := $(build)/test-runner
```

Comment:
all that boilerplate could be handle with proper automake file

Reply:
Agreed. Or with wildcards.